src/components/controls/controls.js

```diff
@@ -2,29 +2,31 @@ import React from "react";
 import { useTranslation } from 'react-i18next';
 
 import ColorBy, {ColorByInfo} from "./color-by";
-import DateRangeInputs, {DateRangeInfo} from "./date-range-inputs";
-import AnimationControls from "./animation-controls";
-import ChooseExplodeAttr from "./choose-explode-attr";
 import ChooseBranchLabelling from "./choose-branch-labelling";
-import ChooseLayout from "./choose-layout";
 import ChooseDataset from "./choose-dataset";
-import ChooseSecondTree from "./choose-second-tree";
 import ChooseTipLabel from "./choose-tip-label";
-import ChooseMetric from "./choose-metric";
 import PanelLayout from "./panel-layout";
 import GeoResolution from "./geo-resolution";
-import TransmissionLines from './transmission-lines';
-import NormalizeFrequencies from "./frequency-normalization";
-import AnimationOptions from "./animation-options";
 import PanelToggles from "./panel-toggles";
-import ToggleTangle from "./toggle-tangle";
 import Language from "./language";
 import { ControlsContainer } from "./styles";
 import FilterData, {FilterInfo} from "./filter";
-import {TreeOptionsInfo, MapOptionsInfo, AnimationOptionsInfo, PanelOptionsInfo,
-  ExplodeTreeInfo, FrequencyInfo, MeasurementsOptionsInfo} from "./miscInfoText";
 import { AnnotatedHeader } from "./annotatedHeader";
-import MeasurementsOptions from "./measurementsOptions";
+import {TreeOptionsInfo, MapOptionsInfo, PanelOptionsInfo} from "./miscInfoText";
+/*, FrequencyInfo, AnimationOptionsInfo, 
+ExplodeTreeInfo,, MeasurementsOptionsInfo*/
+
+//import DateRangeInputs, {DateRangeInfo} from "./date-range-inputs";
+//import AnimationControls from "./animation-controls";
+//import ChooseExplodeAttr from "./choose-explode-attr";
+//import ChooseLayout from "./choose-layout";
+//import ChooseSecondTree from "./choose-second-tree";
+//import ChooseMetric from "./choose-metric";
+//import TransmissionLines from './transmission-lines';
+//import NormalizeFrequencies from "./frequency-normalization";
+//import AnimationOptions from "./animation-options";
+//import ToggleTangle from "./toggle-tangle";
+//import MeasurementsOptions from "./measurementsOptions";
 
 function Controls({mapOn, frequenciesOn, measurementsOn, mobileDisplay}) {
   const { t } = useTranslation();
```

Comment:
This is really great and makes the sidebar a lot less busy. I just had 2 things i wanted to mention. 
1. The APHA logo: i do not think we will need that as we will have this displaying in the nav bar of the app, we could potentially replace this with a back button for the splash page? 
2. Other tree visualisation options: I believe this was removed in an earlier version of the Nextstrain Dashboard after A/B testing, users especially likes the 'radial view' of the tree for reports. I think maybe for the first round of the ALPHA testing we could include this to understand more on this user need?

Reply:
The APHA logo is part of the splash page and clicking on it takes you back to the splash page - so perhaps there's no need for a back button.  For the tree visualizations, my understanding was that it was a single user who liked it?  I think it adds too much confusion for less experienced users

Reply:
Yes - it goes back to the Nextstrain splash page at the moment, but its possible to point auspice to an alternate splash page